ibm_db2/tests/test_unit.py

```diff
@@ -44,3 +48,26 @@ def mock_exception(*args, **kwargs):
             ibmdb2.check(instance)
         # new connection made
         assert ibmdb2._conn != conn1
+
+
+def test_metadata(aggregator, instance, datadog_agent):
+    check = IbmDb2Check('ibm_db2', {}, [instance])
+    check.check_id = CHECK_ID
+
+    check.check(instance)
+
+    # The version string differs from the image tag version
+    raw_version = "DB2 v11.1.2.2"
+    version = raw_version.split("DB2 v")[1]
+
+    major, minor, patch, _ = version.split(".")
```

Comment:
Will the metadata parser drop numbers after patch as well?

Reply:
I was looking into why `build` was failing.

If I understand correctly, the build version format is different from the expected semver regex: https://github.com/DataDog/integrations-core/blob/b3fe21b23fc5f690dfec24ffc1da5e3564efdb9d/datadog_checks_base/datadog_checks/base/utils/metadata/version.py#L20-L22

Do you think I should use a pattern instead of `semver`?

Reply:
That's a good question - might be worth seeing how other integrations that handle non-semver versions.